WhoisthatWebAPI/Controllers/RecognizerController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace WhoisthatWebAPI.Controllers
+{
+    [Route("api/getCascade")]
+    [ApiController]
+    public class RecognizerController : ControllerBase
+
+    {
+
+        [HttpGet]
+        public ActionResult<bool> Execute()
+        {   
+            
+            ExecuteTrainer();
+            return true;
+            
+        }
+        private async Task<bool> ExecuteTrainer()
+        {
+            ProcessStartInfo startInfo = new ProcessStartInfo();
+            startInfo.CreateNoWindow = false;
+            startInfo.UseShellExecute = false;
+            startInfo.FileName = "Trainer.exe";
+            startInfo.WindowStyle = ProcessWindowStyle.Hidden;
+            await StartProcess(startInfo);
+            return true;
+            
+        }
+
+        private async Task StartProcess (ProcessStartInfo startInfo)
+        {
+            try
+            {
+               using (Process exeProcess = Process.Start(startInfo))
+                {
+                   exeProcess.WaitForExit();
+                   
+                }
+            }
+            catch
+            {
+                
```

Comment:
Do not leave catch block empty. What is its purpose then? At least add a comment
```suggestion
                // here will be logging
```